src/presets/cloudflare.ts

```diff
@@ -11,10 +11,13 @@ const cloudflareNodeCompatModules = [
   "_stream_transform",
   "_stream_writable",
   "assert",
+  "assert/strict",
   "buffer",
   "diagnostics_channel",
   "events",
   "path",
+  "path/posix",
```

Comment:
@anonrig has there been any changes on the worked side related to that? I think I tried exactly before messaging you earlier this week and that was not working - could you add a test for i.e. separator